pysen/ext/isort_wrapper.py

```diff
@@ -128,7 +128,7 @@ def run(
     if len(targets) == 0:
         return 0
 
-    cmd = ["isort", "--settings-path", str(setting_path)]
+    cmd = ["isort", "--settings-path", str(setting_path), "--src", str(base_dir)]
```

Comment:
`isort==4` does not seem to support `--src` option. We may need to check isort's version in the same manner to `--recursive` option.

Reply:
`base_dir` may not what you want. it is not the source directory.